### PR TITLE
Updated Spanish translation for 0.7 [#332]

### PR DESCRIPTION
**Pull request by theotocopulitos**
## Status
READY

## Migrations
NO

## Description
Updated Spanish translation for 0.7 [#332]

### CODE REVIEW

File: comixed-frontend/src/assets/i18n/es/backend-status.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
         "detail": "Error al recabar los datos de compilación..."
       }
     },
-    "page-title": "Build Details",
+    "page-title": "Detalles de la compilación",
     "table": {
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/backend-status.json:34]: Can you translate these entries as well?

**Review by mcpierce — CHANGES_REQUESTED** [review]
I saw a few entries that didn't get translated. Could you review the files and fix those entries, please?

For example, in library.json the clear-image-cache entries are still in english.

**Comment by theotocopulitos** [comment]
Sorry :( ... not sure how I missed that...

But, regarding these lines:

    "page-title": "Detalles de la compilación",
    "table": {
      "branch": "Source Branch",
      "build-host": "Build Hostname",
      "build-time": "Built On",
      "build-version": "Version",
      "commit-user": "Committer Name",
      "commit-id": "Commit Hash",
      "commit-message": "Commit Message",
      "commit-time": "Time Of Last Commit",
      "dirty": "Built From Uncommitted Code?",
      "remote-origin-url": "Remote Origin",
      "jdbc-url": "JDBC Connection URL"
    }

I'd rather not translate them... this info relates to git, and there is not
a universal translation of those concepts. Anyone looking at these uses
them in English, I am guessing. Should you prefer me to translate that, I
will, but honestly, I think we should not.

**Comment by mcpierce** [comment]
Fair enough. Do other applications translate similar entries? I believe @BRUCELLA2 and @jmcrfp translated them, but you make a good point about them not being something to translate. Let's leave them untranslated here and we can discuss it in the dev list.

**Comment by theotocopulitos** [comment]
I **think** I was able to squash all changes into a single commit!

**Review by mcpierce — APPROVED** [review]

**Pull request merged**